waspc/src/Generator/Generators.hs

```diff
@@ -66,6 +66,30 @@ generateSrcDir wasp
     ++ EntityGenerator.generateEntities wasp
     ++ Generator.Button.generateButtons wasp
     ++ [generateReducersJs wasp]
+    -- NOTE(matija): for demo only.
+    ++ maybeGenerateXRayDir (getIsXRayModeEnabled wasp)
+
+maybeGenerateXRayDir :: Bool -> [FileDraft]
+maybeGenerateXRayDir isXRayModeEnabled =
+    if isXRayModeEnabled == True then generateXRayDir else []
+
+generateXRayDir :: [FileDraft]
+generateXRayDir = map (simpleCopyFileDraft . (pathToXRayDir </>))
+    [ [relfile|actions.js|] 
+    , [relfile|actionTypes.js|]
+    , [relfile|state.js|]
+    , [relfile|InfoBox.js|]
+    , [relfile|InfoBox.css|]
+    , [relfile|EntityFormInfoBox.js|]
+    , [relfile|EntityListInfoBox.js|]
+    , [relfile|ButtonInfoBox.js|]
+    , [relfile|JsonBrowser.js|]
+    ]
+    where
+        pathToXRayDir = [reldir|src|] </> [reldir|xRay|]
+
+simpleCopyFileDraft :: Path.RelFile -> FileDraft
+simpleCopyFileDraft path = createTemplateFileDraft path path Nothing
```

Comment:
CopyFileDraft is something else, difference between it and template file draft being that copy file draft does not assume source file is positioned in templates. So this name is confusing, if I see simpleCopyFileDraft I assume it is CopyFileDraft, but instead you are hiding templteFileDraft behind. Instead, you should name it `simpleTemplateCopyFileDraft` or `simpleTemplateFileDraft` or `simpleNoDataTemplateFileDraft` or smth like that.

Reply:
aha I see now! Then I shall call it `copyTemplateFileDraft` :D. Is that ok?